kolibri/plugins/management/assets/src/device_management/views/user-permissions-page/index.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+
+  <immersive-full-screen
+    v-bind="{ backPageLink, backPageText }"
+    bodyColorHex="#F9F9F9"
```

Comment:
Not a big fan of this - @khangmach this is related to the styling question about the immersive full screen component above.

Reply:
confirmed, the full-screen modal can simply use this style globally